src/staker.ts

```diff
@@ -0,0 +1,143 @@
+import { BigintIsh, Token, validateAndParseAddress } from '@uniswap/sdk-core'
+import { MethodParameters, toHex } from './utils/calldata'
+import { Interface } from '@ethersproject/abi'
+import { abi } from '@uniswap/v3-staker/artifacts/contracts/UniswapV3Staker.sol/UniswapV3Staker.json'
```

Comment:
can import it from the interface file which is smaller, in case tree shaking doesn't happen